lib/rubocop/cop/rspec/multiple_expectations.rb

```diff
@@ -76,18 +76,39 @@ def on_block(node)
 
         private
 
-        def example_with_aggregated_failures?(node)
-          example = node.send_node
+        def example_with_aggregate_failures?(node)
+          return aggregate_failures?(node) if aggregate_failures_present?(node)
 
-          with_aggregated_failures?(example) &&
-            !disabled_aggregated_failures?(example)
+          parent = parent_block(node)
```

Comment:
Not a big deal, but I guess you can get `node.each_ancestor(:block).find { |parent| aggregate_failures_present?(parent) }` (I feel better naming would be `with_aggregate_failures?`, but then we need to change the matcher name) and then just return `aggregate_failures?(parent) if parent`
Maybe you can move to a method that just returns node (if it defines aggregate failures) or first parent that define, so this just calls aggregate_failures? on the returned node or returns false

Anyway, I'm OK with this code as well, but since this is a `?` method, it should return false, and not nil, if there is no parent defining the metadata

Reply:
Fair enough. The code you suggested is more elegant, loving it.